### ISSUE TITLE
Test accuracy 41% on cora

### ISSUE THREAD

**Issue by qzshadow** [sufeidechabei/graphnets_gcn#1]
Hi Hao,
Thanks for you implementation, I have tested the model accuracy on cora dataset, using the same train/vali/test split by the original tensorflow implementation, however, the accuracy on test set is about 41%, even after I tried to tuning parameters like #layers, #hidden units in each layer, which is somewhat wired compared to original tensorflow implementation (i.g. 81.4% accuracy), have you tested that? Any suggestions?

**Comment by sufeidechabei** [comment]
okay,  I didn't add the adj normalization, I will write it in the message passing process.

**Comment by sufeidechabei** [comment]
Now, you can use our new project. The accuracy is even better than the result in the original paper https://github.com/dmlc/dgl/tree/master/examples/pytorch/gcn

**Comment by qzshadow** [comment]
Thanks, I actually tried to improve your implementation to achieve the reported accuracy in the paper, by add self-loops for the graph, the accuracy is 54% now, which is however still lower than reported accuracy, I am still curious how to implement a correct gcn through graphnets.
